kernelci.org/content/en/docs/org/working-groups.md

```diff
@@ -40,6 +40,8 @@ of what is required.  Based on this, we'll then be looking for technical
 solutions including existing frameworks, dashboards used by other projects and
 potentially a new design from scratch using modern web technology.
 
+The Web Dashboard WG has a [mailing list](https://kernelci.org/docs/org/working-groups/#web-dashboard) at the [address](mailto:<kernelci-webdashboard@groups.io>).
```

Comment:
I'm not sure why "mailing list" links to where it does, given that the user is already reading the section that the link goes to.

I would assume that a link for "mailing list" would link to the actual mailing list?

Reply:
Also, an absolute URL is not a good idea in general.  A relative path would be better - but yes I agree with Chris and this isn't useful here in any case.